### ISSUE TITLE
Reword error message

### ISSUE THREAD

**Issue by markkramerus** [FHIR/sushi#436]
"Cannot resolve element from path:" is a very common error, but users have difficulty understanding the meaning of this message. I suggest a very minor change to reword it to: "The element or path you referenced does not seem to exist:"

**Comment by cmoesel** [comment]
Agree we should reword it.  Not sure about "does not _seem_ to exist" -- the word "seem" feels weird when being stated by a computer program.  Really, unless there is a bug, the element doesn't exist (for real).

Are you ok with one of these?
* The element or path you referenced does not exist: x.y.z
* The referenced path x.y.z does not exist.
* No element found at path: x.y.z

**Comment by markkramerus** [comment]
Of the three, I like the first one.
